blocks/init/src/blocks/components/menu-drawer/menu-drawer-style.scss

```diff
@@ -0,0 +1,71 @@
+$menu-drawer: (
+  width: 256px,
+  colors: (
+    background: #FFFFFF,
+    link: $base-text-color,
+    link-hover: $base-primary-color
+  ),
+  navigation: (
+    font-size: 24px,
+    font-family: $base-font-family
+  ),
+  link-offset: 24px,
+  vertical-padding: 26px,
+  padding-navigation: 16px,
+);
+
+body {
+  .menu-drawer {
+    $menu: &;
+    position: fixed;
+    top: auto;
+    height: 100%;
+    z-index: 99;
+    margin: auto;
+    padding: map-get-deep($menu-drawer, vertical-padding);
+    display: block;
+    background: map-get-deep($menu-drawer, colors, background);
+    width: map-get-strict($menu-drawer, width);
+    transition: transform 0.3s ease-out;
+
+    @include media(desktop up) {
+      display: none;
+    }
+
+    &--left {
+      left: 0;
+      transform: translate3D(-100%, 0, 0);
+    }
+
+    &--right {
+      right: 0;
+      transform: translate3D(100%, 0, 0);
+    }
+  }
+
+  .menu-drawer-nav {
```

Comment:
why not bem?

Reply:
is it because you didn't want to write &-nav or?

Reply:
> why not bem?

Because that should actually be handled by the menu component (just be BEM menu class) and I didn't want to write a new menu component for this specific case.

> is it because you didn't want to write &-nav or?

Yes, that's not really BEM. Tho I can write it like this if you both think this is the better solution.

Reply:
In this component you should create a new component part called menu-drawer-navigation and than use that component that way this component is decoupled